Code/DataGen.py

```diff
@@ -71,7 +71,7 @@ def n_spiral(self, nb_data, noise, num_class):
         features = np.vstack([self.polar_to_cart(radius, angle)]).T
         return features, labels.astype(dtype='int32')
 
-    def generate_data(self, noise, num_class=2):
+    def generates_data(self, noise=0, num_class=2):
         """
         Generate random training and testing sample according to the model name
```

Comment:
I wanted you to correct change "generate" to "generates" only in function description.